rl_algorithms/common/buffer/distillation_buffer.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+"""Distillation buffer."""
```

Comment:
should add detail documentation and reference.
e.g.
https://github.com/medipixel/rl_algorithms/blob/4c6cf558712a940998aca6068c8fb4eaa6b5cd03/rl_algorithms/dqn/agent.py#L1